programs/ziti-edge-tunnel/include/model/dtos.h

```diff
@@ -80,7 +81,7 @@ XX(Addresses, tunnel_address, array, Addresses, __VA_ARGS__) \
 XX(Ports, tunnel_port_range, array, Ports, __VA_ARGS__)  \
 XX(OwnsIntercept, bool, none, OwnsIntercept, __VA_ARGS__) \
 XX(PostureChecks, tunnel_posture_check, array, PostureChecks, __VA_ARGS__) \
-XX(IsAccessible, bool, none, IsAccessible, __VA_ARGS__) \
+XX(IsAccessible, bool, none, IsAccessable, __VA_ARGS__) \
```

Comment:
we should fix Win UI instead of keeping using typos

Reply:
fixed